PhoneBook.Domain/Person.cs

```diff
@@ -90,9 +91,9 @@ public override string DisplayName
       }
     }
 
-    public static Person[] QueryManFindPersonsByLocation (Location location)
+    public static IEnumerable<Person> FindPeopleByLocation (Location location)
     {
-      var query = QueryFactory.CreateQueryFromConfiguration ("QueryManFindPersonsByLocation");
```

Comment:
That identifier is also contained in queries.xml (or what the name of the query xml file is in this solution). it must be changed in both places.